geoprocessing/src/modules/protected-areas/worker/protected-area-worker.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { WorkerBuilder } from '../../worker';
+import { ProtectedAreaProcessor } from './protected-area-processor';
+import { queueName } from './queue-name';
+import { Worker } from 'bullmq';
+
+@Injectable()
+export class ProtectedAreaWorkerService {
+  #worker: Worker;
+
+  constructor(
+    private readonly wrapper: WorkerBuilder,
+    private readonly processor: ProtectedAreaProcessor,
+  ) {
+    this.#worker = wrapper.build(queueName, processor);
+    this.#worker.on('completed', ({ returnvalue }) => {
+      console.log(`---- trigger ApiEvent - ok `, returnvalue);
+    });
+    this.#worker.on('failed', ({ failedReason }) => {
+      console.log(`---- trigger ApiEvent - failure `, failedReason);
+    });
```

Comment:
Hmm, is there a reason to use `console.log` instead of `Logger`, or is it a leftover?

Reply:
@Dyostiq its a placeholder for ApiEvents for next PR

Reply:
Ok, I didn't know :)